src/conf.py

```diff
@@ -101,7 +101,7 @@
     '08-arguments/index.md': '../topics/additional-arguments-and-parameters.html',
     '09-array-inputs/index.md': '../topics/inputs.html',
     '10-array-outputs/index.md': '../topics/outputs.html',
-    '11-records/index.md': '../topics/inputs.html',
+    'inputs/index.md': '../topics/inputs.html',
```

Comment:
? I don't understand.

Reply:
huh. that's odd. I don't remember changing that for this PR. Will revert it, sorry.

Reply:
Ah, the IDE refactor did that change. Reverted.